tools/release/lp-request-import.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/python3
+# This file is part of Checkbox.
+#
+# Copyright 2022 Canonical Ltd.
+# Written by:
+#   Sylvain Pineau <sylvain.pineau@canonical.com>
+#
+# Checkbox is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3,
+# as published by the Free Software Foundation.
+#
+# Checkbox is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Checkbox.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Script that forces a code import from Github in Launchpad.
```

Comment:
I thought it would be nice to link to a page that explains what a "code import" is exactly (I was left wondering if it is a matter of a pull kind of mirroring step or something more elaborate), so I found https://help.launchpad.net/Code/Imports, but it looks like it was last updated 3AD 🙃 Anyhow, it might still be a handy link to include.